### DISCUSSION TITLE
反擊 AI！新工具幫藝術家「毒害」訓練數據

### DISCUSSION THREAD

**Discussion by spermwhale464** [1101-GS4719-C/Python-Community-GS4719-C#1676] [General]
https://technews.tw/2023/10/25/nightshade-can-interfere-with-ai-training-data/
此設計是為了幫助網路藝術家從 AI 公司手中奪回屬於創作權，非常有創意。
其運作原理是在作品檔案加入特別設定的畫素，進而影響AI 訓練，使AI 無法有效辨識內容。

**Comment by Fion-Lin** [comment]
我覺得用這個方式來保護藝術家的權力是一件很酷的事! 但是我覺得生成式AI即使有辦法模擬畫作，但是只有人類才可以富有感情的將畫作呈現出來，這些些微的差異人們還是感受的到的吧~畢竟我目前用AI生成的圖有時候那個整張圖表達的情緒都很怪異，讓人無法理解。

**Comment by mrweixd** [comment]
這其實是個令人放心的消息，畫家終於有了保護自己的手段。活下去的手段、畫風不會被ＡＩ抄走。

**Comment by WeiYen1103** [comment]
隨著生成式AI的發展，有許人利用AI來生成劇本，畫作，以及動畫等等，然後大量放置在網路上，
我認為這樣的方式會使得一些努力創作的小藝術家被這些AI生成的畫作給淹沒，故我覺得應該要有一項規範，來保護這些藝術家們。

**Comment by davidteng00000** [comment]
我覺得如果AI 能與藝術家產出相同質量的照片，那麼其實完全沒有必要為了保護藝術家去做那麼多事情，明顯只是徒增成本與爭議。相反的，應該是藝術家們要去精進他們的著作，產生獨創性，那麼或許被完全取代的時間會延後
